core/controllers/admin.py

```diff
@@ -40,6 +45,17 @@
 import feconf
 import utils
 
+_PARENT_DIR = os.path.abspath(os.path.join(os.getcwd(), os.pardir))
```

Comment:
Why do you need all this stuff here but not in other files?

EDIT: ah, for builtins. Hm. I don't think this is going to work on GAE since oppia_tools will not be available. Instead, can you perhaps get futures imported as a third_party library (so that it gets uploaded to GAE) and then modify the path in appengine_config.py instead?

Reply:
done